pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -474,13 +474,14 @@ class Build {
     // Temurin remote jck trigger
     def remoteTriggerJckTests(String platform) {
         def jdkVersion = getJavaVersionNumber()
-        //def sdkUrl="https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk${jdkVersion}-pipeline/${env.BUILD_NUMBER}/"
-        def filter = "*.tar.gz"
```

Comment:
This could be done by update `def filter = "*-jdk_*.tar.gz" ` and for windows ` filter = "*-jdk_*.zip"`

Reply:
thanks again, yeah I got to here in a roundabout route !

Reply:
used you suggestion